operator/duties/scheduler.go

```diff
@@ -403,9 +389,10 @@ func (s *Scheduler) ExecuteCommitteeDuties(ctx context.Context, logger *zap.Logg
 		if slotDelay >= 100*time.Millisecond {
 			logger.Debug("⚠️ late duty execution", zap.Int64("slot_delay", slotDelay.Milliseconds()))
 		}
-		slotDelayHistogram.Observe(float64(slotDelay.Milliseconds()))
+		slotDelayHistogram.Record(ctx, slotDelay.Seconds())
 		go func() {
 			s.waitOneThirdOrValidBlock(duty.Slot)
+			recordDutyExecuted(ctx, spectypes.BNRoleSyncCommittee)
```

Comment:
this is NOT sync committee, its `Committee` - a special duty that has both attestation and sync committee duties included for the whole slot/committee.

Reply:
good catch!
This metric has `beacon role` as an attribute. I do not think there is Committee role in beacon, since it's more of a SSV term (from what I understand). We also often use `runner role` attributes, which has "committee". Wdyt if we use `runner role` instead of `beacon role` here? I assume scheduler at this point does not know whether it is attestation or sync committee duty

Reply:
There is another option (I think `beacon role` is slightly more valuable attribute(considering our audience) than `runner role`, since runner is our internal term) is to iterate over duties that `spectypes.CommitteeDuty` holds and read `Type` (beacon role) property. That's how we can calculate Attestation and Sync Committee duties and record the metric. What do you think would be better @y0sher ?

Reply:
I think the option to use runner role is probably better. since this metric is about execution, it will be incorrect to say we executed attester duties. we don't execute attester duties anymore, we have our internal committee duty for that. we only submit the attester duties when we finish the committee execution successfully.

Reply:
@y0sher fixed -> changed attribute from `ssv.beacon.role` to `ssv.runner.role`